go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/hofstadter-io/data-utils v0.0.0-20200128210141-0a3e569b27ed
 	github.com/hofstadter-io/dotpath v0.0.0-20191027071558-52e2819b7d2d
 	github.com/hofstadter-io/hof-lang v0.0.0-20180925221847-4d1c962b0729
+	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
```

Comment:
Can we pin this to the most recent semver:
https://github.com/kevinburke/ssh_config/releases/tag/v1.1.0

This version seems out of date